<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sub</class>
 <widget class="QWidget" name="sub">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>540</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>540</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="verticalFrame">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>550</width>
     <height>500</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QFrame" name="horizontalFrame_4">
      <property name="minimumSize">
       <size>
        <width>350</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="text_year">
         <property name="styleSheet">
          <string notr="true">font: 81 12pt &quot;나눔스퀘어 Bold&quot;;</string>
         </property>
         <property name="text">
          <string>0000</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="year_label">
         <property name="styleSheet">
          <string notr="true">font: 81 12pt &quot;나눔스퀘어&quot;;</string>
         </property>
         <property name="text">
          <string>/</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="text_month">
         <property name="styleSheet">
          <string notr="true">font: 81 12pt &quot;나눔스퀘어 Bold&quot;;</string>
         </property>
         <property name="text">
          <string>00</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="month_label">
         <property name="styleSheet">
          <string notr="true">font: 81 12pt &quot;나눔스퀘어&quot;;</string>
         </property>
         <property name="text">
          <string>/</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="text_day">
         <property name="styleSheet">
          <string notr="true">font: 81 12pt &quot;나눔스퀘어 Bold&quot;;</string>
         </property>
         <property name="text">
          <string>00</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="day_label">
         <property name="styleSheet">
          <string notr="true">font: 81 12pt &quot;나눔스퀘어&quot;;</string>
         </property>
         <property name="text">
          <string>/</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="text_week">
         <property name="styleSheet">
          <string notr="true">font: 81 12pt &quot;나눔스퀘어 Bold&quot;;</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QFrame" name="horizontalFrame_1">
      <property name="minimumSize">
       <size>
        <width>540</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>540</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="color_line_2">
         <property name="minimumSize">
          <size>
           <width>280</width>
           <height>7</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>280</width>
           <height>7</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 5px;
border-style: solid;
border-width: 1px;
border-color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QFrame" name="horizontalFrame_2">
      <property name="maximumSize">
       <size>
        <width>540</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="am_or_pm_toggle_button">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 75 10pt &quot;나눔스퀘어 Bold&quot;;
background-color: rgb(255, 255, 255);
border-radius: 5px;
border-style: solid;
border-width: 1px;
border-color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>PM</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="set_time_hours">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="colon_label">
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 75 9pt &quot;나눔바른펜&quot;;</string>
         </property>
         <property name="text">
          <string>:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="set_time_minutes">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QLabel" name="set_task_text_label">
      <property name="minimumSize">
       <size>
        <width>540</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>540</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 9pt &quot;나눔바른펜&quot;;</string>
      </property>
      <property name="text">
       <string>TASK</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QTextEdit" name="set_task_text">
      <property name="minimumSize">
       <size>
        <width>350</width>
        <height>245</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>245</height>
       </size>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QFrame" name="horizontalFrame_3">
      <property name="maximumSize">
       <size>
        <width>540</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QPushButton" name="task_save_button">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 75 10pt &quot;나눔스퀘어 Bold&quot;;
background-color: rgb(255, 255, 255);
border-radius: 5px;
border-style: solid;
border-width: 1px;
border-color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>SAVE</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="see_task_list_button">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 75 10pt &quot;나눔스퀘어 Bold&quot;;
background-color: rgb(255, 255, 255);
border-radius: 5px;
border-style: solid;
border-width: 1px;
border-color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>LIST</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
